src/hooks.js

```diff
@@ -320,6 +320,9 @@ export function useLocalIdentity(entity) {
           return handleRemove(event.addresses)
       }
     })
+
+    updateEntityName()
```

Comment:
Were we not resolving on render without this?

Reply:
Not since the resolving has been moved into this hook, now I’m not sure how it was working in the first place haha